<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionStatusWidget</class>
 <widget class="QWidget" name="ConnectionStatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>64</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelLAN">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../libres/resources/icons.qrc">:/icons/lan.svg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelBluetooth">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../libres/resources/icons.qrc">:/icons/bluetooth.svg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAudio">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../libres/resources/icons.qrc">:/icons/audio.svg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="SignalStrengthWidget" name="widgetSignalStrength" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonConnectionSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../libres/resources/icons.qrc">
       <normaloff>:/icons/settings.svg</normaloff>:/icons/settings.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonRandomizeColor">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Generate a new randomized color for this agent</string>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TryToggle" name="tryToggleConnect" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PanicButton" name="pushButtonPanic">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Panic</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TryToggle</class>
   <extends>QWidget</extends>
   <header location="global">components/TryToggle.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanicButton</class>
   <extends>QPushButton</extends>
   <header location="global">node/PanicButton.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SignalStrengthWidget</class>
   <extends>QWidget</extends>
   <header location="global">connection/SignalStrengthWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../libres/resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonConnectionSettings</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionStatusWidget</receiver>
   <slot>openSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectToggled(bool)</slot>
  <slot>openSettings()</slot>
 </slots>
</ui>
